ccmlib/scylla_repository.py

```diff
@@ -122,3 +119,16 @@ def __get_dir():
     if not os.path.exists(repo):
         os.mkdir(repo)
     return repo
+
+
+def run_scylla_install_script(install_dir, target_dir):
+    scylla_target_dir = os.path.join(target_dir, 'scylla')
+
+    def run(cmd, cwd=None):
+        subprocess.check_call(['bash', '-c', cmd], cwd=cwd, stderr=None, stdout=None)
+
+    run('''sed 's|"$prefix|"$root/$prefix|' -i install.sh''', cwd=install_dir)
```

Comment:
This depends on scylladb/scylla#4949 so we should wait for the fix to get merged rather than doing this patching.

Reply:
If you want this to be merged before scylladb/scylla#4949 is fixed, at least add a comment mentioning the scylla issue and adding a FIXME note to remove this line when the latter is fixed.

Reply:
done